src/tagging/components/tag.jsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+ import style from './style';
+
+class Tag extends React.Component {
+  handleClick = () => this.props.onTagDeleteClick(this.props.tagCategory, this.props.tagValue);
+
+  render() {
+    return (
+      <li key={`${this.props.tagCategory}: ${this.props.tagValue}`}>
+        <span className="label label-info">
+          {this.props.tagCategory}: {this.props.tagValue}
+          <a href="#" onClick={this.handleClick}>
+            <span className="pficon pficon-close" aria-hidden="true" />
```

Comment:
please use the icon component from patternfly-react

Reply:
I will use https://github.com/patternfly/patternfly-react/pull/223 as soon as it's merged